source/app/pages/documents/view.js

```diff
@@ -307,14 +307,20 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
               selected={tab}
               track={track}
               onSelectTab={selectTab}
-              items={[
+              items={ENABLE_COMPREHEND_MEDICAL? [
```

Comment:
This would be preferable
```tsx
const tabItems = [
    { id: 'search', title: 'Preview' },
    { id: 'text', title: 'Raw Text' },
    { id: 'kv', title: `Key-Value Pairs` },
    { id: 'tables', title: `Tables` },
    { id: 'entities', title: `Entities` },
]

if (ENABLE_COMPREHEND_MEDICAL) tabItems.push({ id: 'medical_entities', title: `Medical Entities` })

...

<div className={css.tabWrapper}>
  <Tabs
  isTrackTab={false}
  selected={tab}
  track={track}
  onSelectTab={selectTab}
  items={tabItems}
/>
```